packages/core/src/common/util/migration/migrations/migrateToV8.ts

```diff
@@ -26,7 +26,6 @@ type ExperimentTypeV7 = {
 export const migrateToV8 = (json: ExperimentTypeV7): ExperimentType => {
   return {
     ...json,
-    info: { ...json.info, dataFormatVersion: '8' },
```

Comment:
Shouldn't this line be left in place and the type error solved by returning ExperimentTypeV8 instead? And move ExperimentTypeV8 into this file?

Reply:
Yes, that's better